ivy/functional/backends/jax/experimental/layers.py

```diff
@@ -183,7 +183,7 @@ def general_pool(
         y = jnp.squeeze(y, axis=0)
     return y
 
-
+@with_supported_dtypes({'0.3.25 and below':('float64'),'0.3.25 and above':('float64')}, backend_version)
```

Comment:
I see; there are two problems with this solution
- we still need two lines of spaces between two functions
- and the other when you are writing `{'0.3.25 and below':('float64'),'0.3.25 and above':('float64')}` along with the `@with_supported_dtypes`, you have already included float64 for all the versions. Why did you add it like that?

Thanks!

Reply:
I wanted to make sure that our code supports future versions of JAX and not only 0.3.25 and below which is why I wrote {'0.3.25 and below':('float64'),'0.3.25 and above':('float64')}